plugins/filter_log_to_metrics/log_to_metrics.c

```diff
@@ -729,6 +759,26 @@ static int cb_log_to_metrics_init(struct flb_filter_instance *f_ins,
     }
     ctx->input_ins = input_ins;
 
+
+    /* Initialize timer for scheduled metric updates */
+    sched = flb_sched_ctx_get();
+    if(sched < 0) {
+        flb_plg_error(f_ins, "could not create scheduler");
```

Comment:
Please modify this message, this is not creating a scheduler, it's getting a reference from the thread local storage.